Source/Classes/PINRemoteImageDownloadTask.m

```diff
@@ -139,9 +139,10 @@ - (void)setPriority:(PINRemoteImageManagerPriority)priority
     [super setPriority:priority];
     if (@available(iOS 8.0, macOS 10.10, tvOS 9.0, watchOS 2.0, *)) {
         [self.lock lockWithBlock:^{
-            if (self->_progressImage.dataTask) {
-                self->_progressImage.dataTask.priority = dataTaskPriorityWithImageManagerPriority(priority);
-                [self.manager.urlSessionTaskQueue setQueuePriority:priority forTask:self->_progressImage.dataTask];
+            NSURLSessionDataTask *dataTask = self->_progressImage.dataTask;
+            if (dataTask) {
+                dataTask.priority = dataTaskPriorityWithImageManagerPriority(priority);
+                [self.manager.urlSessionTaskQueue setQueuePriority:priority forTask:dataTask];
```

Comment:
No behavior change here, just cache the data task in a variable for better readability.